Assets/BossRoom/Scripts/Game/Client/State/ClientMainMenuState.cs

```diff
@@ -28,6 +28,7 @@ public class ClientMainMenuState : GameStateBehaviour
         [SerializeField] Button m_LobbyButton;
         [SerializeField] GameObject m_SignInSpinner;
         [SerializeField] UIProfileSelector m_UIProfileSelector;
+        [SerializeField] UITooltipDetector m_UGSSetupTooltipDetector;
```

Comment:
I tested this real quick. Text is really small and takes time to spawn when hovering. If I didn't know it was there, I could easily miss the tooltip.
Text should appear as soon as the mouse is hovering over the button and should adapt to the screen

Reply:
The delay is now removed for this tooltip and I made the text bigger